crates/commons/Cargo.toml

```diff
@@ -6,13 +6,16 @@ edition = "2021"
 [dependencies]
 anyhow = "1"
 bitcoin = { version = "0.30", features = ["serde"] }
+diesel = { version = "2.0.0" }
+r2d2 = { version = "0.8.1" }
```

Comment:
:thinking: Can we try to do this without introducing this dependency here? I guess it's convenient to use `From`, but this is a bit annoying.

Reply:
yes.. I've removed the dependencies and map every error into an anyhow error.